tests/contracts/test_normalization_of_return_types.py

```diff
@@ -28,7 +28,9 @@
                 '0xbb9bc244d798123fde783fcc1c72d3bb8c189413',
             ],
         ),
-    )
+    ),
+    ids=['nullbyte', 'soloaddr', 'addrlist']
```

Comment:
Yes, it fixes the pytest issue, but it also makes the pytest output somewhat more readable:

```
tests/contracts/test_normalization_of_return_types.py::test_normalizing_return_values[nullbyte] PASSED
tests/contracts/test_normalization_of_return_types.py::test_normalizing_return_values[soloaddr] PASSED
tests/contracts/test_normalization_of_return_types.py::test_normalizing_return_values[addrlist] PASSED
```

But I'm only mildly in favor of this approach. If you are more worried about consistency between tests than test output readability for this test, I'm happy to go the version pinning route.